dojo/tools/openvas/xml_parser.py

```diff
@@ -27,7 +28,8 @@ def get_findings(self, filename, test):
                     title = title + "_" + finding.text
                     description.append(f"**Port**: {finding.text}")
                 if finding.tag == "nvt":
-                    description.append(f"**NVT**: {finding.text}")
+                    script_id = finding.get("oid")
+                    description.append(f"**NVT**: {script_id}")
```

Comment:
This will prevent us from showing `finding.text`. If there is any information in the tag's content (`finding.text`), then we could instead print a linebreak and the content (in addition to the `script_id`) to maintain backwards compatibility.